src/AdminDashboard/Settings/Security.js

```diff
@@ -180,65 +184,79 @@ class Security extends Component {
                         </div>
         
                         <div className="listElement">
                             <div className="message">
-                                Kill session
-                                <div className="detail">Destroy the session identified by your session token</div>
+                                {I18n.t('settings.kill_session')}
+                                <div className="detail">
+                                    {I18n.t('settings.destroy_the_session_identified_by_your_session_token')}
+                                </div>
                             </div>
                             <div className="controller">
-                                <button className="win-button" onClick={this.closeSession}>Logout</button>
+                                <button className="win-button" onClick={this.closeSession}>
+                                    {I18n.t('commons.logout')}
+                                </button>
                             </div>
                         </div>
 
                         <Confirmation 
-                            title="Kill session"
-                            message="Are you sure you want to close your session?"
+                            title={I18n.t('settings.kill_session')}
+                            message={I18n.t('logout.are_you_sure_you_want_to_close_your_session?')}
                             reference={el => this.killSession = el} 
                         />
         
                         <div className="listElement">
                             <div className="message">
-                                Delete browser data
-                                <div className="detail">Delete Web Storage and Indexed Database</div>
+                                {I18n.t('settings.delete_browser_data')}
+                                <div className="detail">
+                                    {I18n.t('settings.delete_web_storage_and_indexed_database')}
+                                </div>
                             </div>
                             <div className="controller">
-                                <button className="win-button" onClick={this.cleanWebStorage}>Delete</button>
+                                <button className="win-button" onClick={this.cleanWebStorage}>
+                                    { I18n.t('commons.delete') }
+                                </button>
                             </div>
                         </div>
 
                         <Confirmation 
-                            title="Delete browser data"
-                            message="Are you sure you want to delete all data/information in the web storage and indexed database?"
+                            title={I18n.t('settings.delete_browser_data')}
+                            message={I18n.t('settings.are_you_sure_you_want_to_delete_all_data/information_in_the_web_storage_and_indexed_database?')}
```

Comment:
It think it's too long 😶

Reply:
But this is how transifex needs you to create the key values :face_with_head_bandage: